internal/orchestrion/generator/main.go

```diff
@@ -108,6 +108,11 @@ func generateRootConfig(rootDir string) (map[string]string, error) {
 		return nil, fmt.Errorf("listing packages to import: %w", err)
 	}
 
+	// Add the instrumentation/testutils/grpc/v2 module to the identified modules.
+	// Adding the package to the paths causes issues.
+	instTestutilsGRPC := "github.com/DataDog/dd-trace-go/instrumentation/testutils/grpc/v2"
+	modules[instTestutilsGRPC] = filepath.Join(rootDir, "instrumentation", "testutils", "grpc")
+
```

Comment:
@RomainMuller I had to introduce this "detected" module to fix the CI after pinning all the modules against `v2.1.0-dev`, those having the corresponding `replace` directives.